docs/asciidoc/static/deploying.asciidoc

```diff
@@ -0,0 +1,145 @@
+instance that acts as a server must be configured to use the {logstash}plugins-inputs-lumberjack.html[Lumberjack 
+input plugin].
+
+The Logstash Forwarder uses the computing resources of the machine hosting the source data, and the Lumberjack input 
+plugin minimizes the resource demands on the Logstash instance, making this architecture attractive for use cases with 
+resource constraints.
+
+image::images/deploy_3.png[]
+
+// [float]
+[[deploying-larger-cluster]]
+=== Scaling to a Larger Elasticsearch Cluster
+
+Typically, Logstash does not communicate with a single Elasticsearch node, but with a cluster made up of several nodes. 
+Logstash can use any of the protocols Elasticsearch makes available to move data into the cluster: 
+{guide}_transport_client_versus_node_client.html[HTTP, transport, or node]. 
+
+You can use the HTTP REST APIs that Elasticsearch provides to index data into the Elasticsearch cluster, transporting 
+the data in a JSON representation. Using the REST APIs does not require the Java client classes or any additional JAR 
+files and has no performance disadvantages compared to the transport or node protocols. You can secure communications 
+that use the HTTP REST APIs with the {shield}[Shield] plugin, which supports SSL and HTTP basic authentication.
+
+You can also use the Java APIs that Elasticsearch provides to serialize the data into a binary representation, using 
+the transport protocol. The transport protocol has the option to sniff the endpoint of the request, selecting an 
+arbitrary client or data node in the Elasticsearch cluster. 
+
+Both the HTTP and transport protocols separate Logstash from the Elasticsearch cluster. The node protocol, by contrast, 
+has the machine running the Logstash instance join the Elasticsearch cluster, running an Elasticsearch instance. The 
+data that needs indexing propagates from this node to the rest of the cluster. Since the machine is part of the 
+cluster, the cluster topology is available, making the node protocol a good fit for use cases that use a relatively 
+small number of persistent connections.
+
+When you use the HTTP protocol, you can configure the Logstash Elasticsearch output plugin to automatically 
+load-balance indexing requests across a 
+{logstash}plugins-outputs-elasticsearch.html#plugins-outputs-elasticsearch-host[specified set of hosts] in the 
+Elasticsearch cluster. Specifying multiple Elasticsearch nodes also provides high availability for the Elasticsearch 
+cluster by routing traffic to active Elasticsearch nodes.
+You can also use a third-party load balancer, either hardware or software, to handle connections between Logstash and 
+external applications.
+
+NOTE: Make sure that your Logstash configuration does not connect directly to Elasticsearch 
+{ref}modules-node.html[master nodes]. Master nodes tend to have substantial demands on their computing resources. 
```

Comment:
> Master nodes tend to have substantial demands on their computing resources

This is incorrect: masters are not resource heavy. Its just that master nodes have a dedicated role to orchestrate the cluster activities, so you don't want to overwhelm it with indexing requests. Can we rephrase this?